docs/src/docs/usage/configuration.mdx

```diff
@@ -54,3 +54,9 @@ than the default and have more strict settings:
 ```yaml
 { .GolangciYaml }
 ```
+
+## Cache
+
+GolangCI-Lint stores his cache into the [default user cache directory](https://golang.org/pkg/os/#UserCacheDir).
```

Comment:
```suggestion
GolangCI-Lint stores its cache in the [default user cache directory](https://golang.org/pkg/os/#UserCacheDir).
```

Definitely not "his", and probably not "into" (but I'm not a native speaker too).